widgets/BigQueryTable-batchsource.json

```diff
@@ -4,6 +4,35 @@
   },
   "display-name": "BigQuery",
   "configuration-groups": [
+    {
+      "label": "Connection",
+      "properties": [
+        {
+          "widget-type": "toggle",
+          "label": "Use connection",
+          "name": "useConnection",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
the widget-type for this should be `connection-select`

Reply:
done:)